csharp/ql/test/library-tests/assignables/Assignables.expected

```diff
@@ -82,6 +82,17 @@
 | Assignables.cs:109:16:109:16 | s |
 | Assignables.cs:115:13:115:13 | i |
 | Assignables.cs:116:13:116:13 | s |
+| Assignables.cs:120:36:120:36 | i |
+| Assignables.cs:120:36:120:36 | i |
+| Assignables.cs:120:36:120:36 | i |
+| Assignables.cs:120:36:120:36 | i |
```

Comment:
Why are these duplicated?

Reply:
Yeah, that is a bit tricky, but it is because each delegate parameter exists in 4 copies; one for the delegate type itself, one for the `Invoke` method, and one each for `BeginInvoke` and `EndInvoke`.